Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationAdministrativeTemplatePolicyWindows10/MSFT_IntuneDeviceConfigurationAdministrativeTemplatePolicyWindows10.psm1

```diff
@@ -0,0 +1,1707 @@
+        #endregion
+
+        $nullResult = $PSBoundParameters
+        $nullResult.Ensure = 'Absent'
+
+        $getValue = $null
+        #region resource generator code
+        $getValue = Get-MgDeviceManagementGroupPolicyConfiguration -GroupPolicyConfigurationId $Id -ErrorAction SilentlyContinue
+
+        if ($null -eq $getValue)
+        {
+            Write-Verbose -Message "Could not find an Intune Device Configuration Administrative Template Policy for Windows10 with Id {$Id}"
+
+            if (-Not [string]::IsNullOrEmpty($DisplayName))
+            {
+                $getValue = Get-MgDeviceManagementGroupPolicyConfiguration `
+                    -Filter "DisplayName eq '$DisplayName'" `
+                    -ErrorAction SilentlyContinue
+            }
+        }
+        #endregion
+        if ($null -eq $getValue)
+        {
+            Write-Verbose -Message "Could not find an Intune Device Configuration Administrative Template Policy for Windows10 with DisplayName {$DisplayName}"
+            return $nullResult
+        }
+        $Id = $getValue.Id
+        Write-Verbose -Message "An Intune Device Configuration Administrative Template Policy for Windows10 with Id {$Id} and DisplayName {$DisplayName} was found."
+
+        #region resource generator code
+        $enumPolicyConfigurationIngestionType = $null
+        if ($null -ne $getValue.PolicyConfigurationIngestionType)
+        {
+            $enumPolicyConfigurationIngestionType = $getValue.PolicyConfigurationIngestionType.ToString()
+        }
+        #endregion
+
+        #region
+        $settings = Get-MgDeviceManagementGroupPolicyConfigurationDefinitionValue `
+            -GroupPolicyConfigurationId $Id
+
+        $complexDefinitionValues = @()
+        foreach ($setting in $settings)
+        {
+            $definitionValue = @{}
+            $definitionValue.Add('Id', $setting.Id)
+            if ($null -ne $setting.ConfigurationType)
+            {
+                $definitionValue.Add('ConfigurationType', $setting.ConfigurationType.toString())
+            }
+            $definitionValue.Add('Enabled', $setting.Enabled)
+            $definition = Get-MgDeviceManagementGroupPolicyConfigurationDefinitionValueDefinition `
+                -GroupPolicyConfigurationId $Id `
+                -GroupPolicyDefinitionValueId $setting.Id
+
+            $enumClassType = $null
+            if ($null -ne $definition.ClassType)
+            {
+                $enumClassType = $definition.ClassType.ToString()
+            }
+
+            $enumPolicyType = $null
+            if ($null -ne $definition.PolicyType)
+            {
+                $enumPolicyType = $definition.PolicyType.ToString()
+            }
+            $complexDefinition = @{
+                CategoryPath = $definition.CategoryPath
+                ClassType    = $enumClassType
+                DisplayName  = $definition.DisplayName
+                #ExplainText           = $definition.ExplainText
```

Comment:
Why are these lines commented out? Can't they be removed altogether? The schema does have these parameters.

Reply:
@ykuijs just removed those commented parameters
Most of those are added as read only just to give some context on the export